utils/meta.go

```diff
@@ -0,0 +1,58 @@
+package utils
+
+import (
+	"time"
+)
+
+type (
+	// Meta description
+	Meta struct {
+		Name      string      `json:"name"`
+		Timestamp int64       `json:"timestamp"`
+		Value     interface{} `json:"value"`
+	}
+
+	// Metas describe all meta
```

Comment:
// Metas contains all metadata for the collector itself, the configured source and configured sink